.sonarqube/sonar-scanner.properties

```diff
@@ -6,7 +6,6 @@ sonar.modules= asn1_codec, \
     jpo-ode-core, \
     jpo-ode-plugins, \
     jpo-ode-svcs, \
-    jpo-s3-deposit, \
```

Comment:
Should we add the jpo-utils project in here?

Reply:
It appears that SonarQube is primarily for java/c based projects and may not be useful for jpo-utils as it is primarily docker image customization rather than raw source code. This may be something that we can discuss further with Leidos when we merge into the primary USDOT codebase.